<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskTrackerEntryClass</class>
 <widget class="QWidget" name="TaskTrackerEntryClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>72</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TaskTrackerEntry</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,1,1,3,1">
     <item>
      <widget class="QLabel" name="IndexLabel">
       <property name="text">
        <string>1.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="StartField">
       <property name="text">
        <string>09:00</string>
       </property>
       <property name="placeholderText">
        <string>Start time</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="EndField">
       <property name="text">
        <string/>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>End time</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="DurationField">
         <property name="text">
          <string/>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>Duration</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="DurationHoursField">
         <property name="text">
          <string/>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>Hours</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLineEdit" name="DescriptionField">
       <property name="placeholderText">
        <string>Description</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="EndButton">
       <property name="text">
        <string>End</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>EndButton</sender>
   <signal>clicked()</signal>
   <receiver>TaskTrackerEntryClass</receiver>
   <slot>OnEndButtonPress()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>551</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>35</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>EndField</sender>
   <signal>textChanged(QString)</signal>
   <receiver>TaskTrackerEntryClass</receiver>
   <slot>OnEndFieldModified(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>35</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>StartField</sender>
   <signal>textChanged(QString)</signal>
   <receiver>TaskTrackerEntryClass</receiver>
   <slot>OnStartFieldModified(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>35</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DescriptionField</sender>
   <signal>textChanged(QString)</signal>
   <receiver>TaskTrackerEntryClass</receiver>
   <slot>OnDescriptionFieldModified(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>390</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>35</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnEndButtonPress()</slot>
  <slot>OnEndFieldModified(QString)</slot>
  <slot>OnStartFieldModified(QString)</slot>
  <slot>OnDescriptionFieldModified(QString)</slot>
 </slots>
</ui>
